Tests/Command/ParatestCommandTest.php

```diff
@@ -20,15 +20,15 @@
  * Tests/App/AppKernel.php.
  * So it must be loaded in a separate process.
  *
- * @runTestsInSeparateProcesses
+ * @runInSeparateProcess
```

Comment:
Please revert this, this annotation is only for functions: https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.runInSeparateProcess

It may fix the error with the failing test.

Reply:
Reverted